security/enforce-policy-istio.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Enforce network policy using Istio
+title: Enforce Calico network policy using Istio
```

Comment:
Kubernetes network policies will also be enforced.  So I think the original wording is more accurate.

Reply:
Fixed